### PR TITLE
Grades Trend

### PR DESCRIPTION
**Pull request by MykolaGalian**
[US123099](https://rally1.rallydev.com/#/23525809118d/iterationstatus?detail=%2Fuserstory%2F462188715396&sharedViewId=328666453148&fdp=true?fdp=true): [Engagement][UserDrill] Grades Trend

With zoom

First PR with test data.
With a flag that hides this stuff for now (on for Demo and tests only).

Functional:
Local App work as expected with test data.
Browsers (Windows 10 Chrome, Firefox, Edge)

Accessibility: n/a
Security: n/a
Devops: n/a
UI: in progress

Next PR - Course colors for cards

FYI @johngwilkinson @anhill-D2L @bemailloux

### CODE REVIEW

File: components/grades-card.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
+import 'highcharts';
+import { css, html } from 'lit-element/lit-element.js';
+import { BEFORE_CHART_FORMAT } from './chart/chart';
+import { bodyStandardStyles } from '@brightspace-ui/core/components/typography/styles';
+import { Localizer } from '../locales/localizer';
+import { MobxLitElement } from '@adobe/lit-mobx';
+import { SkeletonMixin } from '@brightspace-ui/core/components/skeleton/skeleton-mixin';
+
+class GradesCard extends SkeletonMixin(Localizer(MobxLitElement)) {
```
Comment (Vitalii-Misechko) [inline components/grades-card.js:9]: `suggestion` GradesCard -> GradesTrendCard

Comment (Vitalii-Misechko) [reply components/grades-card.js]: Thank you for the change. Could you update also the filename and langterm names?

File: components/grades-trend-card.js [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+		// any change to a relevant observed property of the Data object
+		return html`
+			<div class="d2l-insights-grades-trend-title d2l-skeletize d2l-skeletize-45 d2l-body-standard">${this._cardTitle}</div>
+			<d2l-labs-chart class="d2l-insights-summary-card-body"
+							.options="${this.chartOptions}"
+							?skeleton="${this.skeleton}"
+			></d2l-labs-chart>`;
+	}
+
+	get chartOptions() {
```
Comment (NicholasHallman) [inline components/grades-trend-card.js:87]: you can use the colors in ./const once you merge master, the course-legend component has an example of using the iterator to loop through the colors endlessly

Comment (MykolaGalian) [reply components/grades-trend-card.js:87]: Ok, I will add this in the next PR

**Review by Vitalii-Misechko — APPROVED** [review]

**Review by NicholasHallman — APPROVED** [review]

**Pull request merged**